chromadb/db/mixins/sysdb.py

```diff
@@ -185,6 +186,7 @@ def create_collection(
         self,
         id: UUID,
         name: str,
+        configuration: CollectionConfiguration,
```

Comment:
shouldn't this be the json str? Otherwise the grpc API is wrong?

Reply:
I think it's correct, see GRPC impl higher in the stack. GRPC doesn't use the SysDB interface directly.